src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -59,7 +60,7 @@ export const ShortTerm = ({
                                                 <Donut
                                                     key={index}
                                                     qualifying={qualifying}
-                                                    periodicPayment={periodicPayment}
+                                                    periodicPayment={formattedPeriodicPayment}
```

Comment:
```suggestion
                                                    periodicPayment={periodicPayment.replace(/(\s?EUR)/g, ' €')}
```
Can we change this to the above? it might get confusing if we call it `formattedPeriodicPayment` since that's an actual variable that exists as far as content coming back is concerned. Also, instead of wrapping it in `currencyFormat` which also lops off 0s, we probably just want to have the above for now just to keep the 0s since we don't like removing those when it's a payment breakdown. This regex above will only update EUR to the symbol and everything else stays the same.